mlos_bench/db/db_schema.sql

```diff
@@ -50,48 +173,29 @@ CREATE TABLE experiment_merge (
         REFERENCES trial(exp_id, trial_id)
 );
 
--- Tunable parameters and their types, used for merging the data from several experiments.
--- Records in this table must match the `mlos_bench.tunables.Tunable` class.
-CREATE TABLE tunable_params (
-    param_id VARCHAR(255) NOT NULL,
-    param_type VARCHAR(32) NOT NULL,  -- One of {int, float, categorical}
-    param_default VARCHAR(255),
-    param_meta VARCHAR(1023),  -- JSON-encoded metadata (range or categories).
-
-    PRIMARY KEY (param_id)
-);
-
-CREATE TABLE trial_config (
-    exp_id VARCHAR(255) NOT NULL,
-    trial_id INTEGER NOT NULL,
-    param_id VARCHAR(255) NOT NULL,
-    param_value VARCHAR(255),
-
-    PRIMARY KEY (exp_id, trial_id, param_id),
-    -- TODO: Enable when we pre-populate the `tunable_params` table:
-    -- FOREIGN KEY (param_id) REFERENCES tunable_params(param_id),
-    FOREIGN KEY (exp_id, trial_id) REFERENCES trial(exp_id, trial_id),
-    FOREIGN KEY (exp_id) REFERENCES experiment(exp_id)
-);
-
-CREATE TABLE trial_results (
+-- A table to store final trial result scores (if trial status != FAILED).
+-- Note: each trial may have multiple scores (e.g. one for each objective).
+-- Each trial score must be associated with a metric_id from the corresponding output_metrics table.
+CREATE TABLE trial_result (
     exp_id VARCHAR(255) NOT NULL,
     trial_id INTEGER NOT NULL,
-    metric_id VARCHAR(255) NOT NULL,
-    metric_value VARCHAR(255),
+    metric_id INTEGER NOT NULL,
```

Comment:
Changed this one to an integer reference to the other table for consistency, but if we drop the `output_metrics` table could certainly just make this a VARCHAR again, but I'd probably call it `metric_name` then instead to allow for the possibility of adding these other linking tables in the future.